gateway/1.4/modules/ROOT/pages/_partials/task-config-logs-third-party.adoc

```diff
@@ -1,10 +1,21 @@
 //tag::intro1[]
-Anypoint Flex Gateway enables you to configure log output for consumption by third-party services. Logs can be delivered to any supported Fluent Bit {fluentbit-ver-var} output. You configure log output via a custom YAML file.
+Anypoint Flex Gateway enables you to deliver runtime and access logs to any supported Fluent Bit {fluentbit-ver-var} output type. You configure log output via a custom YAML file. Additionally, to view access logs, you must also apply a Message Logging policy.
+
+The following configuration file examples demonstrates how to direct Flex Gateway logs to output types: `File`, `HTTP`, and `Splunk`. However, you can use any Fluent Bit {fluentbit-ver-var} output type, such as `Azure Log Analytics`, `Amazon S3`, `Kafka`, and more. For a complete list of Fluent Bit output types, see https://docs.fluentbit.io/manual/v/1.8/pipeline/outputs[Fluent Bit Outputs documentation^]. 
```

Comment:
We should remove links to the versioned Fluent Bit documentation. It presents a maintenance issue for us when we support a new version of Fluent Bit. Can we link instead to the Fluent Bit homepage? Or do we need the link at all?

Reply:
You could put it in the See Also list.